internal/deployer/workloads.go

```diff
@@ -0,0 +1,74 @@
+// Package deployer for project deployment
+package deployer
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"strings"
+
+	"github.com/threefoldtech/grid3-go/workloads"
+	"github.com/threefoldtech/zos/pkg/gridtypes"
+	"github.com/threefoldtech/zos/pkg/gridtypes/zos"
+)
+
+func (d *Deployer) deployNetwork(ctx context.Context, networkName, projectName string, node uint32) error {
+	network := workloads.ZNet{
+		Name:  networkName,
+		Nodes: []uint32{node},
+		IPRange: gridtypes.NewIPNet(net.IPNet{
+			IP:   net.IPv4(10, 20, 0, 0),
+			Mask: net.CIDRMask(16, 32),
+		}),
+		SolutionType: projectName,
+	}
+	return d.tfPluginClient.NetworkDeployer.Deploy(ctx, &network)
+
+}
+func (d *Deployer) deployVM(ctx context.Context, networkName, projectName, repoURL, vmName string, node uint32) error {
+
+	vm := workloads.VM{
+		Name:       vmName,
+		Flist:      "https://hub.grid.tf/aelawady.3bot/abdulrahmanelawady-gridify-test-latest.flist",
+		CPU:        2,
```

Comment:
I think we shouldn't make constant values inside the functions. please check all